src/Arcus.Testing.Tests.Integration/Integration/DataFactory/Fixture/TemporaryDataFactoryDataFlow.cs

```diff
@@ -315,9 +351,12 @@ public async Task UploadToSourceAsync(string expected)
                 _ => throw new ArgumentOutOfRangeException()
             };
 
-            _logger.LogTrace("Upload {FileType} file to DataFlow source: {FileContents}", _dataType, expected);
+            string filePath = subPath.Aggregate(SourceDataSetName, Path.Combine);
```

Comment:
Good use of `.Aggregate`! (Or ‘making it foldable’, like we functional programmers say 😄).